src/common/paths/dbrps.yml

```diff
@@ -7,10 +7,14 @@ get:
     - $ref: "../parameters/TraceSpan.yml"
     - in: query
       name: orgID
-      required: true
```

Comment:
Most endpoints allow specifying either the `orgID` or `org` (name), so the org name parameter was added in several places and the `orgID` required designation was removed. The "one or the other" requirement is handled in the application code.